homeassistant/components/stream/recorder.py

```diff
@@ -78,7 +78,13 @@ async def async_record(self) -> None:
 
         def write_segment(segment: Segment) -> None:
             """Write a segment to output."""
-            nonlocal output, output_v, output_a, last_stream_id, running_duration, last_sequence
+            nonlocal \
+                output, \
+                output_v, \
+                output_a, \
+                last_stream_id, \
+                running_duration, \
+                last_sequence
```

Comment:
Ouch, for me the use of `\` is a breaking deal here 😅

Reply:
Would that `nonlocal (..., ..., ...)` was valid syntax 😄 

That said, maybe the use of `nonlocal` here is something to look at in general; it smells like these should be instance variables :)